src/connectionManager.js

```diff
@@ -701,12 +701,16 @@ export default class ConnectionManager {
             });
         };
 
-        function onSuccessfulConnection(server, systemInfo, connectionMode, serverUrl, options, resolve) {
+        function onSuccessfulConnection(server, systemInfo, connectionMode, serverUrl, options={}, resolve) {
             const credentials = credentialProvider.credentials();
-            options = options || {};
+
             if (options.enableAutoLogin === false) {
                 server.UserId = null;
                 server.AccessToken = null;
+            } else if (server.AccessToken && options.enableAutoLogin !== false) {
```

Comment:
You have already checked this.
```suggestion
            } else if (server.AccessToken) {
```